feature-libs/quote/components/details/cart/quote-details-cart.component.spec.ts

```diff
@@ -191,6 +248,22 @@ describe('QuoteDetailsCartComponent', () => {
     });
   });
 
+  describe('onToggleShowOrHideCart', () => {
+    it('should call quoteDetailsCartComponentService correctly if argument is true', () => {
+      component.onToggleShowOrHideCart(true);
```

Comment:
What about `toggleCardView()`?

Reply:
I would rather not change it in this PR as not related. I only added 2 tests because I saw the test coverage was poor (and I think that is a quite minor issue)